GitUI/CommandsDialogs/FormRebase.cs

```diff
@@ -11,16 +11,17 @@ namespace GitUI.CommandsDialogs
     public partial class FormRebase : GitExtensionsDialog
     {
         #region Mnemonics
-        // Available: GHJLNVWXYZ
+        // Available: GHJLVWXYZ
         // A Add files
         // B Abort
         // C Continue rebase
         // D Ignore date
-        // E Specific range
+        // E Update dependent refs
         // F From
         // I Interactive
         // K Skip
         // M Committer date
+        // N Specific range
```

Comment:
Have you forgot to update the regarding strings in designer and translation?

Reply:
No I don't think so (except if I really missed something). I have just updated the comment because the mnemonic was not accurate for "Specific range" (which is "n" in the UI) and so use the free "e".